002_simulation_structure/002_simulation_structure.tex

```diff
@@ -1,23 +1,109 @@
+ 
+The tower is made up of multiple floors, positively incrementing as the depth of the floor increases. On the top floor, Floor Zero, there is a kitchen which produces a fixed amount of food per day, and on each floor below this, the people of the tower reside. 
+The floor on which people are based is random, and periodically reshuffles with a fixed frequency. At the beginning of each day a platform begins uniformly moving down the tower, starting from the top floor where it is loaded with food and stopping for a constant time at each floor. Over a period of time, there is enough food on the platform to minimally satisfice the agents, but not maximally satisfy them all.
+ 
+The people exist with limited knowledge of tower, they must eat enough within a period to survive and can talk with other individuals on local floors. They do know which floor number they are on, but do not know how many floors there are in the tower nor how much food there is at the beginning of each day. Additionally, the people in the tower do not explicitly know how often the reshuffle period is. Finally, when a person dies, they are immediately replaced.
+
+The scenario presented above is heavily influenced by \textit{'El Hoyo'}\citep{elhoyo2019} (translating directly to 'The Hole') a Spanish Movie released in 2019.
+
+\subsection{Assumptions}
+
+
+There are several features that can be assumed from the given context and, for the implementation in this paper, are described in this section.
+
+\begin{enumerate}
+    \item  Pre-existing knowledge – people exist in the tower environment with a level of knowledge of the environment, this reduces the amount of learning required before action can take place. Pre-existing ideologies are included in this, people have their ways of interacting with others, views on socialising and the like; perhaps acquired from a life before the tower.
+     Basic concepts such as consuming food increases health are known
+      Learning what the world is vs learning to apply in the world.
+      \item  Food Taking – people are allowed to take as much food from the platform as they are able to eat. They cannot preserve food to eat on separate days. Additionally, the utility of the food taken decreases exponentially (each unit of food has a decreasing improvement to HP) – representing the law of diminishing returns and acting as a deterrent to basic strategies.
+      \item A sufficiently honest environment – people are generally truthful, direct deceit does not exist, only the ability to ignore. There are very specific conditions in which deceit is permitted, these will be discussed alongside the introduction of the relevant features, but is related to breaking agreements in times of extreme need.
```

Comment:
```suggestion
      \item An honest environment -- people are assumed to be truthful: direct deceit does not exist. There are very specific conditions in which deceit is permitted, these will be discussed alongside the introduction of the relevant features, but is related to breaking agreements in times of extreme need.
```
When is deceit allowed??

Reply:
I think team 7 LMAO, they will break a treaty to eat and survive.....

Reply:
I think they just misunderstood Team 5's code when they said that

Reply:
LMAO - should I remove that part then?